setup.py

```diff
@@ -27,9 +27,11 @@ def package_data(pkg, roots):
     version='0.2',
```

Comment:
Please bump the version number -- we'll tag this once PR is merged too.

Reply:
@swalladge  Thanks for bumping the version here!  Could you create a tag for the merged commit, named to match the new `0.2.1` version?

Reply:
Done :)